Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm

```diff
@@ -339,9 +339,8 @@ String resolvePathForSandboxExtension(StringView path)
 auto SandboxExtension::createHandlesForMachLookup(Span<const ASCIILiteral> services, std::optional<audit_token_t> auditToken, MachBootstrapOptions machBootstrapOptions, OptionSet<Flags> flags) -> Vector<Handle>
 {
     auto handles = createHandlesForResources(services, [auditToken, flags] (ASCIILiteral service) -> std::optional<Handle> {
-        auto handle = createHandleForMachLookup(service, auditToken, flags);
-        ASSERT(handle);
-        return handle;
+        // Note that createHandleForMachLookup() if the process has just crashed.
```

Comment:
I think you meant to say "returns null" in this comment?

Reply:
Yes, will fix.